share/functions/help.fish

```diff
@@ -78,6 +78,11 @@ function help --description 'Show help for the fish shell'
 		return 1
 	end
 
+        # In Cygwin, start the user-specified browser using cygstart
+        if begin; type -q cygstart; and test $fish_browser != "cygstart"; end
```

Comment:
It was failing with fish v2.2.0, so I followed the doc's recommendations on how to do multi-command tests.